tests/positive/files/units.go

```diff
@@ -320,3 +321,57 @@ func RemoveEnablementSymLinksforUnit() types.Test {
 		ConfigMinVersion: configMinVersion,
 	}
 }
+
+// testPresetsFileinAlphabeticalOrder checks if Ignition
+// writes the systemd presets in alphabeitcal order by
+// unit name into the 20-ignition.preset file.
+func testPresetsFileInAlphabeticalOrder() types.Test {
+	name := "unit.presets.alphabetical.order"
```

Comment:
The dots form a notional hierarchy of tests, and we probably won't have another test under `unit.presets.alphabetical`.  Maybe something like `unit.presets.sorted`?

Reply:
I took a wild guess on the naming convention here. Thanks for the feedback, it's been updated.